profile/2018-12-01-profile.json

```diff
@@ -0,0 +1,623 @@
+                           "dnszones/NS",
+                           "dnszones/CAA",
+                           "dnszones/recordsets",
+                           "dnszones/all"
+                         ],
+                         
+                         "2018-08-01": [
+                             "applicationGatewayAvailableSslOptions",
+                             "applicationGatewayAvailableWafRuleSets",
+                             "applicationGateways",
+                             "applicationSecurityGroups",
+                             "bgpServiceCommunities",
+                             "connections",
+                             "checkTrafficManagerNameAvailability",
+                             "ddosProtectionPlans",
+                             "expressRouteCircuits",
+                             "expressRouteServiceProviders",
+                             "interfaceEndpoints",
+                             "loadBalancers",
+                             "localNetworkGateways",
+                             "locations",
+                             "locations/availableDelegations",
+                             "locations/bareMetalTenants",
+                             "locations/CheckDnsNameAvailability",
+                             "locations/checkAcceleratedNetworkingSupport",
+                             "locations/effectiveResourceOwnership",
+                             "locations/operationResults",
+                             "locations/operations",
+                             "locations/supportedVirtualMachineSizes",
+                             "locations/setResourceOwnership",
+                             "locations/usages",
+                             "locations/validateResourceOwnership",
+                             "locations/virtualNetworkAvailableEndpointServices",
+                             "networkIntentPolicies",
+                             "networkInterfaces",
+                             "networkProfiles",
+                             "networkSecurityGroups",
+                             "networkWatchers",
+                             "networkWatchers/connectionMonitors",
+                             "networkWatchers/pingMeshes",
+                             "operations",
+                             "publicIPAddresses",
+                             "publicIPPrefixes",
+                             "routeFilters",
+                             "routeTables",
+                             "serviceEndpointPolicies",
+                             "trafficmanagerprofiles",
+                             "trafficManagerGeographicHierarchies",
+                             "virtualNetworkGateways",
+                             "virtualNetworks",
+                             "virtualNetworkTaps"
+                         ]
+        },
+
+        "microsoft.notificationHubs": {
+
+                         "2016-03-01": [
+                             "operationResults",

```

Comment:
`operationResults` endpoint is not defined in microsoft.notificationHubs `2016-03-01` see [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/notificationhubs/resource-manager/Microsoft.NotificationHubs/stable/2016-03-01/notificationhubs.json).